common/app/views/fragments/inlineSvg.scala.html

```diff
@@ -1,4 +1,12 @@
 @(file: String, path: String = "", classes: Seq[String] = List(), title: Option[String] = None)
+
+@import views.support.setSvgClasses
+
 <span @title.map{ t => title="@t" } class="inline-@file @if(path){inline-@path} @classes.mkString(" ")">
-    @Html(common.Assets.inlineSvg(path + "/" + file + ".svg"))
+    @defining((
+        path + "/" + file + ".svg",
+        classes ++ Seq("inline-" + file, "inline-" + path)
```

Comment:
They will get `inline-@file__svga and `inline-@path__svg`: https://github.com/guardian/frontend/pull/13336/files/75a1c3ba7dbd31553bd583ee9402797e80f5f91e#diff-a78830a0e574fa7224f8e492f8ac2c9aR601